go/libraries/doltcore/sqle/database.go

```diff
@@ -693,6 +693,11 @@ func (db Database) getTableInsensitive(ctx *sql.Context, head *doltdb.Commit, ds
 		return dt, found, nil
 	}
 
+	if resolve.UseSearchPath && lwrName == doltdb.RebaseTableName {
```

Comment:
Might add a comment about what this is doing (converting `dolt_rebase` into `dolt.rebase` for compatibility